cves/kernel/CVE-2021-4154.yml

```diff
@@ -324,8 +329,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: The commit was signed off by Christian Brauner (the author) and Linus Torvalds (the committer).
```

Comment:
Not sure if we should mention names directly? I asked this in another comment https://github.com/VulnerabilityHistoryProject/vulnerabilities/pull/42#discussion_r1002561717

Reply:
I will be changing that thanks!